packages/ckeditor5-dev-translations/lib/utils/getsourcemessages.js

```diff
@@ -30,3 +30,14 @@ export default function getSourceMessages( { packagePaths, sourceFiles, onErrorC
 			return sourceMessages;
 		} );
 }
+
+/**
+ * @typedef {object} Message
```

Comment:
```suggestion
 * @typedef {object} TranslatableEntry
```

WDYT?

Reply:
Done in https://github.com/ckeditor/ckeditor5-dev/pull/1025/commits/5eb675f3874e4aefc83807afcfe12c182521eccd.